client/components/share/tumblr-share-preview/index.jsx

```diff
@@ -1,71 +1,36 @@
-import { localize } from 'i18n-calypso';
+import { TumblrFullPreview } from '@automattic/social-previews';
 import { PureComponent } from 'react';
-import { truncateArticleContent } from '../helpers';
-
-import './style.scss';
+import { decodeEntities } from 'calypso/lib/formatting';
 
 export class TumblrSharePreview extends PureComponent {
 	render() {
 		const {
 			externalProfilePicture,
-			externalProfileUrl,
+			externalProfileURL,
 			externalName,
-			message,
 			articleUrl,
 			articleTitle,
 			articleContent,
 			imageUrl,
-			translate,
+			message,
 		} = this.props;
 
-		const summary = truncateArticleContent( 396, articleContent );
+		const username = externalProfileURL?.match( /[^/]+$/ )[ 0 ];
+
 		return (
-			<div className="tumblr-share-preview">
```

Comment:
All this markup is not relevant anymore. Preview is implemented in the `social-previews` package.